tox.ini

```diff
@@ -144,5 +144,12 @@ description = Build the full documentation
 
 extras = docs
 
+setenv =
+    # Set this to `True` to run similar to Read the docs.
+    READTHEDOCS=False
+    READTHEDOCS_LANGUAGE=en
+    # Possible version: 1.2.3 stable latest
```

Comment:
We don't actually support `stable` at the moment, I think.

Does RTD literally pass `stable` in `READTHEDOCS_VERSION` or does it pass the release tag instead?

Reply:
It literally pass "stable".

My plan is to also support stable and any other tag as part of #330